src/main/resources/init_db.sql

```diff
@@ -9,3 +9,66 @@ CREATE TABLE `taxi_service`.`manufactures` (
 
   INSERT INTO `taxi_service`.`manufactures` (`name`, `country`) VALUES ('tesla', 'usa');
```

Comment:
in this file, on line 4 - change `manufacture_id` to `id`

Reply:
fixed